push_notifications/services/zeropush.py

```diff
@@ -16,30 +18,39 @@ def __init__(self, settings):
             raise ImproperlyConfigured('For ZeroPush to work you need an '
                                        'AUTH_TOKEN in the configuration')
 
+    @staticmethod
+    def process_expiry(expiry):
+        if isinstance(expiry, datetime):
+            expiry = expiry.second
+
+        if isinstance(expiry, timedelta):
+            expiry = expiry.seconds
+        return expiry
+
     def send_push_notification(self, devices, message,
                                badge_number=None, sound=None,
                                payload=None, expiry=None):
         if len(devices):
             params = {
                 "auth_token": self.auth_token,
-                "device_tokens[]": [device.token for device in devices],
-                "expiry": expiry,
-                "sound": sound,
-                "info": payload
+                "device_tokens[]": [device.token for device in devices]
             }
 
-            for key in params.keys():
-                if not params[key]:
-                    del params[key]
+            if message is not None:
```

Comment:
Why did we go back to the `if` statements instead of leaving the implementation of deleting it from the dict if the key doesn't exist?